src/components/ChefOfTheWeekSection/ChefOfTheWeekSection.tsx

```diff
@@ -1,43 +1,84 @@
-import React from "react";
 import { SectionTitle } from "../SectionTitle/SectionTitle";
-import "./ChefOfTheWeekSection.scss";
+import styles from "./ChefOfTheWeekSection.module.scss";
+import ChefsRestaurantCard from "../ChefsRestaurantCard/ChefsRestaurantCard";
+import { Chef } from "../../types/types";
+import data from "../../data/data.json";
+import useGetScreenWidth from "../../hooks/useGetWidthScreen";
+import { UIConstants } from "../../shared/constants";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+
+import goToIcon from "../../assets/images/goToIcon.svg";
 
 export const ChefOfTheWeekSection = () => {
+  const chefs: Chef[] = data.data.chefs;
+  const chefOfTheWeek: Chef | undefined = chefs.find(
```

Comment:
Note that writing out the type `Chef | undefined` is a bit redundant as it is inferred automatically.
Just so you know.

Reply:
@nitzanpap  done